Automattic/wp-calypso
# Checkout: Remove RefundTerms100Year component and file

This is a follow up to https://github.com/Automattic/wp-calypso/pull/84125

This PR cleans up the call to RefundTerms100Year and removes the component file altogether. The 100 Year Plan refund policy is now handled by the refund-policies.tsx file.

Related to #84125 

## Testing Instructions

* Add a 100 Year Plan to your cart by using this url format - `https://wordpress.com/checkout/[your_site_url]/wp_bundle_hundred_year`, if using a Calypso Live link then just change the portion after `/checkout`
* Ensure only one instance of the following exists - `You will be charged C$52,920 and understand that [refunds](https://wordpress.com/support/refunds/) are limited to 120 days after purchase.`

**Incorrect**

<img width="678" alt="image" src="https://github.com/Automattic/wp-calypso/assets/16580129/ad1db78d-ff7e-4b50-bc47-5f5d0f26786e">


**Correct**

<img width="650" alt="image" src="https://github.com/Automattic/wp-calypso/assets/16580129/2f0ce62a-64e2-4d10-8dcf-921c8fa6fec6">
